.expeditor/verify_linux_pipeline.yml

```diff
@@ -0,0 +1,56 @@
+  - label: "[@@plan@@] :linux: :habitat: Check for bad patterns"
```

Comment:
How do we feel about moving the linux and windows pipeline YAML files, since they aren't technically valid pipeline YAMLs, into a templates directory, or with a tpl suffix? 

```
.expeditor/templates/verify_linux_pipeline.yml
```

OR

```
.expeditor/verify_linux_pipeline.yml.tpl
```

I personally prefer the first.

Reply:
Also, Buildkite doesn't have a `:habitat:` emoji. They only have a `:habicat` emoji. https://github.com/buildkite/emojis

Reply:
Good catch,  fixed and moved to `.expeditor/templates/`